assessmentModel/src/commonMain/kotlin/serialization/InputItems.kt

```diff
@@ -12,25 +12,20 @@ import org.sagebionetworks.assessmentmodel.survey.AnswerType
 val inputItemSerializersModule = SerializersModule {
     polymorphic(InputItem::class) {
         subclass(CheckboxInputItemObject::class)
-        subclass(DateInputItemObject::class)
```

Comment:
Shouldn't TimeInputItemObject be here?

Reply:
It was already in the model. This list is alphabetized.